Sources/OAWorldRegion.mm

```diff
@@ -621,16 +621,9 @@ - (OAWorldRegion *) findAtLat:(double)latitude lon:(double)longitude
 
 - (NSString *) getCountryNameAtLat:(double)latitude lon:(double)longitude
 {
-    NSArray<OAWorldRegion *> *list = [self queryAtLat:latitude lon:longitude];
-    for (OAWorldRegion *region in list)
-    {
-        if ([region contain:latitude lon:longitude])
-        {
-            NSString *name = region.name;
-            if (name && name.length > 0)
-                return name;
-        }
-    }
+    OAWorldRegion *region = [self findAtLat:latitude lon:longitude];
```

Comment:
Is this correct? It changes the logic of the method greatly. Will this break other places where this code is used?
/